tutorial/lesson_21_auto_scheduler_generate.cpp

```diff
@@ -0,0 +1,192 @@
+// Halide tutorial lesson 21: Auto-Scheduler
+
+// This lesson demonstrates how to use the auto-scheduler to generate a
```

Comment:
Prefix this comment with something like: 

// So far we have written Halide schedules by hand, but it is also possible to ask Halide to suggest a reasonable schedule. We call this auto-scheduling. This lesson demonstrates ...